packages/wrangler/pages/functions/routes.ts

```diff
@@ -114,7 +114,7 @@ export function parseConfig(config: Config, baseDir: string) {
     });
   }
 
-  for (const [route, props] of Object.entries(config.routes)) {
+  for (const [route, props] of Object.entries(config.routes ?? {})) {
```

Comment:
Wont this break horrifically if it tries to parse an empty object?

Reply:
Why? If `config.routes` is an empty object, then `Object.entries(config.routes) === []`, so the `for` loop will never execute. `routes` will continue to be an empty array, as defined on L71.